package.json

```diff
@@ -21,26 +21,33 @@
     "identity-obj-proxy": "^3.0.0",
     "jest": "^23.5.0",
     "prop-types": "^15.6.2",
+    "classnames": "^2.2.6",
+    "match-sorter": "^2.3.0",
+    "moment": "^2.22.2",
     "react": "^16.4.2",
     "react-bootstrap": "^0.32.3",
     "react-dom": "^16.4.2",
     "react-redux": "^5.0.1",
     "react-router": "^3.0.0",
     "react-router-dom": "^4.3.1",
     "react-test-renderer": "^16.4.2",
+    "react-table": "^6.8.6",
     "redux": "^3.6.0",
     "redux-form": "^7.4.2",
-    "redux-saga": "^0.16.0"
+    "redux-promise-middleware": "^5.1.1",
+    "redux-saga": "^0.16.0",
+    "redux-thunk": "^2.3.0"
```

Comment:
I think there is no need to add thunk here as we are already using Saga.

Reply:
Sounds good!

Reply:
Although right now we can assume we are making a single REST call to the order endpoint to fetch a list of orders, once we receive the list, I think we will need to make a call-per-row to determine the specimen collection data (if any) for each row in the order list.  This seems like something redux-saga would be good for?  

Alternatively, we could come up with something on the backend to return everything we need in a single request...